wsi-test-helper/package.json

```diff
@@ -2,24 +2,23 @@
   "name": "wsi-test-helper",
   "version": "1.0.0",
   "description": "Helpers for webpack-subresource-integrity tests",
-  "main": "index.js",
+  "main": "dist/index.js",
   "license": "MIT",
   "private": true,
   "devDependencies": {
-    "@tsconfig/node10": "^1.0.8",
+    "@tsconfig/node12": "^1.0.9",
```

Comment:
We should update Node in this project too, sooner or later, I guess 😺

Reply:
It's the minimum version and no need to update it at this point.